src/attached_probe.h

```diff
@@ -30,7 +30,8 @@ class AttachedProbe
                 BpfProgram &&prog,
                 int pid,
                 BPFfeature &feature,
-                BTF &btf);
+                BTF &btf,
+                bool safe_mode = false);
```

Comment:
Why default to non safe mode?

Reply:
This default isn't being used by anything checking safe_mode so fine to make it default to true.